.github/workflows/static-code-analysis.yml

```diff
@@ -38,3 +38,17 @@ jobs:
 
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v1
+
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name:
+      uses: ikerexxe/lintly-flake8-github-action@fix_execution
+      if: github.event_name == 'pull_request'
+      with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          failIf: new
+          args: "--ignore=E121,E123,E126,E226,E24,E704,W503,W504,W605 --max-line-length=119 ."
```

Comment:
* `--max-line-length=119` can be removed so it rely on the configuration file
* this is a diff-scan, right? Do we really want to ignore anything? I randomly looked up these errors but all seems to be worth a fix.

Reply:
You are right. I just copied those from our current ignore list without processing the list. I have reviewed them and almost all of them should be checked. My only concern is regarding [W503](https://www.flake8rules.com/rules/W503.html) and [W504](https://www.flake8rules.com/rules/W504.html). Isn't one the contrary of the other? Taking into account that W504 seems to be the good one, I'll only leave W503 in the ignore list.

Reply:
I agree.

Reply:
Just add it to the configuration instead of the command line.

Reply:
Done!